### DISCUSSION TITLE
"Cheval-vapeur" in French is equal to what in English?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#431] [Galaxy Trivia] [answered]
**Bonus:** How is it calculated?

**Comment by alirezazahiri** [comment, marked as answer]
"Cheval-vapeur" is a term used to express horsepower in the metric system in French-speaking countries. It is often abbreviated as "ch" or "CV". In English, it is typically referred to as "horsepower" (hp).

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/alirezazahiri?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Comment by alirezazahiri** [comment]
> How is it calculated?

One horsepower is defined as the amount of power required to lift 550 pounds (or 250 kilograms) at a rate of one foot (or 0.3048 meters) per second.

In metric units, one horsepower is equivalent to 735.5 watts. Therefore, to convert horsepower (hp) to watts (W), you can use the following formula:

1 hp = 735.5 W

Conversely, to convert watts (W) to horsepower (hp), you can use the following formula:

1 W = 0.00136 hp

**Comment by thuongtruong1009** [comment]
"Cheval-vapeur" is a French term that is often abbreviated as "CV." In English, "cheval-vapeur" translates to "horsepower" (hp). It is a unit of power that is commonly used to express the power output of engines, particularly in automobiles.
